app/services/service_provider_updater.rb

```diff
@@ -1,9 +1,10 @@
 class ServiceProviderUpdater
   PROTECTED_ATTRIBUTES = [
-    :id,
+    :approved,
```

Comment:
@monfresh what was the reason for making `approved` a protected attribute here? I think this may have resulted in a bug where we can't create new approved SPs from the dashboard app